gulpfile.js

```diff
@@ -23,18 +23,31 @@ var injectVersion = require('gulp-inject-version');
 var jshint = require('gulp-jshint');
 var minifyCss = require('gulp-clean-css');
 var minifyHtml = require('gulp-minify-html');
+var rename = require("gulp-rename");
+var rollup = require('gulp-rollup');
 var uglify = require('gulp-uglify');
 var vulcanize = require('gulp-vulcanize');
 
 gulp.task('default', ['vulcanize', 'copy-files']);
 
-gulp.task('copy-files', ['clean'], function() {
+gulp.task('bundle', function() {
+  gulp.src('./src/**/*.js')
+    // transform the files here.
+    .pipe(rollup({
+      // any option supported by Rollup can be set here.
+      entry: './src/main.js'
+    }))
+    .pipe(rename("all.js"))
```

Comment:
Thinking out loud......we could potentially exclude "index.js" from vulcanize. Then, instead of having this rolllup as a separate task, we could pipe the result of vulcanize and crisper to rollup. Then it's all one pipeline with no temporary "all.js".